bindings/resolve.go

```diff
@@ -26,6 +26,36 @@ import (
 // Predicate should return true if it matches a given binding.
 type Predicate func(bind libcnb.Binding) bool
 
+// And combines multiple predicates logical and-ing their results together.
+func And(predicates ...Predicate) Predicate {
```

Comment:
Do we need this one? `And` is the default behavior when multiple Predicates are provided.

Reply:
Nope, fair enough.